Dockerfile.centos7

```diff
@@ -0,0 +1,20 @@
+FROM centos:7
+
+RUN yum update -y && yum install -y centos-release-ansible-27 centos-release-scl-rh && \
+    yum -y install ansible && \
+    yum -y install epel-release && \
+    PKGS="rh-git218 python3-pip standard-test-roles python2-fmf seabios-bin" && \
+    yum -y install $PKGS && \
+    rpm -V $PKGS && \
+    yum clean all && \
+    pip3 install cachecontrol
+
+RUN useradd -m tester
+USER tester
+
+COPY test /test
+
+VOLUME /config /secrets /cache
+
+WORKDIR /home/tester
+CMD ["scl", "enable", "rh-git218", "/test/run-tests"]
```

Comment:
The Fedora container spec uses dumb-init to ensure that the container can properly receive signals/handles zombie processes. This seems to be missing here.

Reply:
> The Fedora container spec uses dumb-init to ensure that the container can properly receive signals/handles zombie processes. This seems to be missing here.

Not sure where to get dumb-init from for centos7.  Also, it seems to work without it - not sure what I'm missing here, or how to tell if not using it will cause problems.  Do you have a link to the Fedora container spec which talks about the use of dumb-init?

Reply:
The README in https://github.com/Yelp/dumb-init explains it. Without it, there will be a lot of zombie processes in OpenShift - at least this is what we saw in the past.

Reply:
yep - a lot of zombie processes - doesn't make sense to me - does run-tests need to store handles to the child processes in a global array, use a SIGCHLD handler, and call `wait` on each child process in the SIGCHLD handler?  In all my years of working on openshift I have never seen this - but then none of the projects I worked on had long running container commands that did a lot of fork/exec . . .
Added back dumb-init to save debugging cycles . . .

Reply:
Sorry, I don't remember the details but I guess it is related to the qcow2 inventory script forking, so the processes are not direct child processes of run-test.